src/cart/crt_init.c

```diff
@@ -316,8 +316,7 @@ crt_init_opt(crt_group_id_t grpid, uint32_t flags, crt_init_options_t *opt)
 	if (gdata_init_flag == 0) {
 		rc = data_init(server, opt);
 		if (rc != 0) {
-			D_ERROR("data_init failed, rc(%d) - %s.\n",
-				rc, strerror(rc));
+			D_ERROR("data_init failed " DF_RC "\n", DP_RC(rc));
```

Comment:
It was decide a few weeks ago (talk to Jeff) that there shouldn't be spaces around DF_RC and like functions.  Jeff has been changing these in most files and I have changed it in my files.

Reply:
done